test/parallel_api/iterator/permutation_iterator_parallel_transform_reduce.pass.cpp

```diff
@@ -56,8 +56,9 @@ DEFINE_TEST_PERM_IT(test_transform_reduce, PermItIndexTag)
                     dpl::copy(exec, permItBegin, permItEnd, sourceData.begin());
                     wait_and_throw(exec);
 
-                    const auto expected = ::std::transform_reduce(sourceData.begin(), sourceData.end(), TestValueType{},
-                                                                  ::std::plus<TestValueType>(), ::std::negate<TestValueType>());
+                    const auto expected = TestUtils::transform_reduce_serial(sourceData.begin(), sourceData.end(),
```

Comment:
@danhoeflinger sould you please reformat this code?

Reply:
done